tests/integration/components/frost-list-test.js

```diff
@@ -64,12 +64,12 @@ describeComponent(
       const $localHook = $hook
 
       return wait().then(() => {
```

Comment:
Instead of a const for the hook, does it work if you pass in $hook here - `then(($hook) => {}` ?